cognite/neat/_data_model/exporters/_table_exporter/exporter.py

```diff
@@ -49,34 +49,50 @@ def export(self, data_model: RequestSchema) -> DataModelTableType:
                     row[prop] = None
         return output
 
-    def as_excel(self, data_model: RequestSchema, file_path: Path, options: WorkbookOptions | None = None) -> None:
-        """Exports the data model as an Excel file.
+
+class DMSYamlExporter(DMSTableExporter):
+    """Exports DMS to YAML."""
+
```

Comment:
My thinking is that YAMl should always exclude None as that is more efficient storage. 

Your refactoring actually makes this very nice if you add this

```suggestion
    """Exports DMS to YAML."""

    def __init__(self) -> None:
        super().__init__(exclude_none=True)

```